src/shared/components/query/QueryStore.spec.ts

```diff
@@ -122,4 +124,92 @@ describe("QueryStore", ()=>{
             // },1000)  
         });
     });
-});
\ No newline at end of file
+});
+describe("QueryStore", ()=>{
+    describe("Volcano Plot related functions", ()=>{
+        const genesets: Geneset[] = [{
```

Comment:
see mocha documentation for use of before and beforeEach for data "set up"  this is a good habit to get into because for more involved tests which might involve state or instances of classes, it's very important to prevent tests from interfering with each other